### DISCUSSION TITLE
Image background

### DISCUSSION THREAD

**Discussion by iainhallam** [combostrap/combo#64] [Q&A]
Hi, folks. Is there a way I can set an image as a page background, to cover the whole page? Looks as though I can set something in CSS on the body element, but then would need to also give `#page-core` or `#main-content` a background so the text can be separated from the background. Sadly this leaves a gap below the header which is margin rather than padding.

**Comment by gerardnico** [comment]
The only way is to create a new theme:

https://combostrap.com/howto/how-to-create-your-own-theme-m092ki81

**Comment by gerardnico** [comment]
Additionally, you can also use the dokuwiki css user style if a thème is too much.
